ddtrace/_trace/trace_handlers.py

```diff
@@ -691,6 +691,28 @@ def _on_botocore_patched_bedrock_api_call_success(ctx, reqid, latency, input_tok
     span.set_tag_str("bedrock.usage.completion_tokens", output_token_count)
 
 
+def _on_rq_queue_enqueue_job_is_async(ctx, job):
+    span = ctx[ctx["call_key"]]
```

Comment:
Is ctx always expected to have call_key. Do we need to check if the key exists and then log a warning/error if it doesn't. Ideally we should never crash a customers application

Reply:
@emmettbutler  from what I see `call_key` isn't really being verified to exist in other handlers..it just is expected to exist. Is there an integration example where this is not included? 

I've been using `span=ctx[ctx["call_key"]]` to inject the  span.context or add span tags...is there an alternative way of doing this without call_key?

Reply:
`ctx["call"]` accesses the [default](https://github.com/DataDog/dd-trace-py/blob/e692716e8211b50555066e18c17285d4c45c4985/ddtrace/_trace/trace_handlers.py#L121) attribute that stores the span on contexts that have the [start_span](https://github.com/DataDog/dd-trace-py/blob/e692716e8211b50555066e18c17285d4c45c4985/ddtrace/_trace/trace_handlers.py#L852) handler attached to them. This is slightly more "safe" than using `call_key`, but only marginally so because both access data set conditionally elsewhere in the code. `if ctx.get_item("call", None) is not None` is the safest way to access a possibly-extant span stored on an `ExecutionContext`.